server/migrations/20211001211616-create-comment.ts

```diff
@@ -0,0 +1,27 @@
+"use strict";
+module.exports = {
+  up: async (queryInterface: any, Sequelize: any) => {
+    await queryInterface.createTable("PostComments", {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.UUID,
+        unique: true
+      },
+      body: {
+        type: Sequelize.TEXT,
+      },
```

Comment:
```suggestion
      body: {
        type: Sequelize.TEXT,
        allowNull: false,
      },
```
We can also perform a non-empty check using hooks or a regex.